### DISCUSSION TITLE
Undefined variable $endpoint

### DISCUSSION THREAD

**Discussion by retrobot-gists** [tzsk/payu#256] [Q&A]
I get error "Undefined variable $endpoint", all other variables from view get undefined as well.

From form view:
 <div style="display: none;">
        <form action="" id="payment_form" method="POST">
            @foreach($fields as $key => $value)
                <input type="hidden" name="{{ $key }}" value="{{ $value }}">
            @endforeach
        </form>
    </div>

    <script>
        document.getElementById('payment_form').submit();
    </script>

**Comment by tzsk** [comment]
> I get error "Undefined variable $endpoint", all other variables from view get undefined as well.
> 
> From form view:
> 
> @foreach($fields as $key => $value) @Endforeach
> ```
> <script>
>     document.getElementById('payment_form').submit();
> </script>
> ```

Please specify the PHP Version, Laravel Version & Payu Package Version.

**Reply by NazishDeveloper** [reply]
I am having the same issue I am using laravel 8 and PHP 8.1.4
